src/interfaces/GeoChart.ts

```diff
@@ -18,6 +18,25 @@ export interface IPushpinColor {
     background: string;
 }
 
+export type IGeoCenter = [number, number];
```

Comment:
We can re-use https://docs.mapbox.com/mapbox-gl-js/api/#lnglatlike

Reply:
updated, change logic, so It's not necessary